scripts/generate-post-data.sh

```diff
@@ -0,0 +1,29 @@
+# Generates JSON using a bash function - See https://stackoverflow.com/a/57369772/2391795
+# "End Of File" must be at the beginning of the line with no space/tab before or after - See https://stackoverflow.com/a/12909284/2391795
```

Comment:
outdated doc

can some doc be added regarding EndOfMessage? I don't know what's that

Reply:
@Demmonius Ping

Reply:
done